workspaces/bazaar/plugins/bazaar-backend/dev/index.ts

```diff
@@ -19,7 +19,7 @@ const backend = createBackend();
 
 backend.add(import('@backstage/plugin-auth-backend'));
 backend.add(import('@backstage/plugin-auth-backend-module-guest-provider'));
-backend.add(import('@backstage/plugin-catalog-backend/alpha'));
-backend.add(import('../src/alpha'));
+backend.add(import('@backstage/plugin-catalog-backend'));
+backend.add(import('../src/plugin'));
```

Comment:
Shouldn't this just be `../src`? For example: https://github.com/backstage/community-plugins/blob/fd649d15e67d4d25e9036dbdb1623d729bb2858f/workspaces/airbrake/plugins/airbrake-backend/dev/index.ts#L22

Reply:
Yes it should. Thank you.